test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityControllerSpec.scala

```diff
@@ -6169,15 +6169,13 @@ class YearToDateLiabilityControllerSpec
 
       "show a form error" when {
 
-        def test(
-          sessionData: SessionData
-        )(
+        def test(sessionData: SessionData)(
           data: (String, String)*
         )(expectedTitleKey: String, expectedErrorKey: String, expectedErrorArgs: String*): Unit =
           testFormError(data: _*)(
             expectedErrorKey,
             expectedErrorArgs
-          )(expectedTitleKey)(performAction, sessionData)
+          )(expectedTitleKey, expectedErrorArgs: _*)(performAction, sessionData)
```

Comment:
a bit hacky - but it's not possible to pass multiple var arg arguments in

Reply:
another option would be to have
```scala
(expectedTitleKey: String, expectedTitleArgs: String*)(expectedErrorKey: String, expectedErrorArgs: String*)
```
with the currying